error.go

```diff
@@ -183,3 +183,11 @@ func (e *ErrNoPointerEvents) Unwrap() error {
 func (e *ErrNoPointerEvents) Is(err error) bool {
 	return reflect.TypeOf(e) == reflect.TypeOf(err)
 }
+
+// ErrPageNotFound error
+type ErrPageNotFound struct {
```

Comment:
Please implement the `Is(err error) bool` interface: https://golang.org/pkg/errors/#Is